packages/react-components/src/components/RemoteVideoTile.tsx

```diff
@@ -111,6 +114,20 @@ export const _RemoteVideoTile = React.memo(
         /* @conditional-compile-remove(one-to-n-calling) */
         /* @conditional-compile-remove(PSTN-calls) */
         participantState={props.participantState}
+        isPinned={props.isPinned}
+        onClickPinIcon={() => {
+          props.onUnpinParticipant?.();
+        }}
+        menuItems={[
```

Comment:
if we don't add these conditionally its kinda a breaking change - Contoso will now have a pin button that doesn't work.
Consider only including the menu items if `onPinParticipant` is provided?